jibri/rootfs/etc/cont-init.d/10-config

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/with-contenv bash
+export JIBRI_INSTANCE_ID=$(date +%N)
+
+# set external IP as xmpp domain
+echo "${DOCKER_HOST_ADDRESS}	${XMPP_DOMAIN}" >> /etc/hosts
+echo "${DOCKER_HOST_ADDRESS}	${XMPP_SERVER}" >> /etc/hosts
+
+# check loaded snd_aloop module
+[[ $(lsmod |grep -om1 snd_aloop) ]] \
+|| ( echo -e "\e[31mERROR: Please load snd-aloop module on the docker host.\e[39m"; kill 1; exit 1 )
+
+# always recreate configs
+frep --overwrite /defaults/config.json:/etc/jitsi/jibri/config.json
+frep --overwrite /defaults/logging.properties:/etc/jitsi/jibri/logging.properties
+
+# make recording dir
+mkdir -p /tmp/recordings
```

Comment:
Put this in /config so they are accessible from the outside.

Reply:
I think it would be a better idea to put this in another directory like /recordings for multiple reasons.
1. It's not a configuration
2. You can separate the volumes to use ConfigMaps and persitent storage in kubernetes environments

Reply:
@sapkra its already changed. Please see 48286a3

Reply:
use `$JIBRI_RECORDING_DIR` and `$JIBRI_LOGS_DIR` for adjust paths.